tools/buildutils/cw/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable AS base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update -y && apt upgrade -y
+RUN apt install -y sudo devscripts
+
+FROM base AS with_bazel
+
+RUN --mount=source=.,target=/mnt,type=bind \
+  /mnt/tools/buildutils/installbazel.sh
+
+FROM with_bazel
```

Comment:
what's the purpose of using multiple `FROM` instructions here?

Reply:
Removed those statements. Added them based on a miss assumptions on how cache works when building images.